src/dmgbuild/licensing.py

```diff
@@ -382,87 +158,119 @@
     ),
 }
 
-
-class LPicResource(Resource):
-    def __init__(self, res_id, res_name, default_lang, lpic, res_attrs=0):
-        data = []
-        data.append(struct.pack(b'>HH', default_lang, len(lpic)))
-        for lang, rid, two_byte in lpic:
-            data.append(struct.pack(b'>HHH', lang, rid, int(two_byte)))
-        super().__init__(b'LPic', res_id, res_name, b''.join(data), res_attrs)
-
-
-def get_encoder_name(locale):
-    if locale not in region_codes:
-        raise Exception("Cannot determine region code for locale '%s'" % locale)
-    region_code = region_codes[locale]
-
-    if region_code not in script_codes:
-        raise Exception("Cannot determine script code for locale '%s'" % locale)
-    script_code = script_codes[region_code]
-
-    if script_code not in encodings_map:
-        raise Exception("Cannot determine Python encoder name for locale '%s' - "
-                        "encode the string data manually as a byte array instead" % locale)
-    return encodings_map[script_code]
+ufifrezXMLtemplate = {
+    'LPic': [
+        {
+            'Attributes': '0x0000',
+            'Data': b'\x00\x00\x00\x02\x00\x00\x00\x00\x00\x00\x00\x00\x00\x04\x00\x00',
+            'ID': '5000',
+            'Name': ''
+        }
+    ],
+    'STR#': [
+        {
+            'Attributes': '0x0000',
+            'Data': b'\x00\x06\rEnglish test1\x05Agree\x08Disagree\x05Print\x07Save...zIMPORTANT - Read this License Agreement carefully before clicking on the "Agree" button. By clicking on the "Agree" button, you agree to be bound by the terms of the License Agreement.',  # noqa: E501
+            'ID': '5000',
+            'Name': 'English buttons'
```

Comment:
Is "English buttons" intentional here? It doesn't seem consistent with the block that is added at L259.

Reply:
I'm wondering if this block is needed at all - it seems to be a duplicate of what will be added when the English language is processed; except for the details of the "Data" block, and the "English buttons" vs "English" label.

Reply:
`English buttons` is maybe a left over from a found [xml template](https://github.com/create-dmg/create-dmg/blob/master/support/eula-resources-template.xml#L36) will change it to `English`

Reply:
not sure to remove this block, if no language is set in the settings this will be the fallback